src/app/retrospective/sprint-task-summary/sprint-task-summary.component.ts

```diff
@@ -64,27 +64,26 @@ export class SprintTaskSummaryComponent implements OnInit, OnChanges, OnDestroy
     }
 
     ngOnChanges(changes: SimpleChanges): void {
+        let refreshData = true;
         if (changes.isTabActive) {
             this.isTabActive = changes.isTabActive.currentValue;
+        } else if (changes.enableRefresh) {
```

Comment:
please verify if changes always has just one key or not. If not remove if-else logic

Reply:
Also, check every other `ngOnChanges` for the similar pattern